frontend/src/Components/Navbar/NavbarSamfThree.module.scss

```
@@ -0,0 +1,326 @@
+  overflow: hidden;
+  transform: scaleY(0);
+  transform-origin: 50% 0%;
+  opacity: 0;
+}
+
+.dropdown_container_left .dropdown_container {
+  right: 0;
+}
+
+.dropdown_open {
+  pointer-events: all;
+  overflow: visible;
+  transform: scaleY(1);
+  opacity: 1;
+}
+
+.navbar_dropdown_link,
+.navbar_logout_button {
+  padding: 0.5rem 1rem;
+  text-decoration: none;
+  color: $white;
+  cursor: pointer;
+  font-size: inherit;
+  display: flex;
+  align-items: center;
+  gap: 0.3rem;
+  background: none;
+  border: none;
+
+  @include theme-dark {
+    color: $white;
+  }
+
+  &:hover {
+    background: $navbar-dropdown-link-dark-hover;
+    text-decoration: none;
+  }
+}
+
+/* Desktop navigation items */
+.navbar_main_links {
+  display: flex;
+  justify-content: space-between;
+  height: 100%;
+  margin-right: auto;
+  font-size: 16px;
+  font-family: $navbar-font-family;
+  text-transform: uppercase;
+}
+
+.navbar_main_links_mobile {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.navbar_link {
+  color: inherit;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  padding: 0 1rem;
+  height: 100%;
+  cursor: pointer;
+  flex: 1;
+  text-align: center;
+  justify-content: center;
+  z-index: 2;
+  font-size: inherit;
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  @include for-tablet-down {
+    display: none;
+  }
+
+  // This ensures navbar dropdown shadow does not lie
+  // on top of the navbar item
+  &.selected {
+    background-color: $navbar-bg-dark;
+  }
+
+  /* Link hover for regular navbar */
+}
+
+/* Samf logo */
+.navbar_logo {
+  display: flex;
+  text-decoration: none;
+  margin: 0 1rem;
+  width: 250px;
+  cursor: pointer;
+}
+
+#navbar_logo_img {
+  width: 100%; // QUESTION: Why is this needed?
+  padding: 13px;
+}
+
+/* Widgets with language, theme, login */
+.navbar_widgets {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  margin-right: 24px;
+  margin-left: 24px;
+  gap: 15px;
+  height: 100%;
+
+  @include for-tablet-down {
+    display: none;
+  }
+}
+
+/* Container for profile */
+.navbar_profile_button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  white-space: nowrap;
+  cursor: pointer;
+  padding-left: 6px;
+  padding-right: 6px;
+  background-color: inherit;
+  gap: 5px;
+
+  @include for-desktop-up {
+    height: 100%;
+  }
+
+  .profile_icon {
+    width: 16px;
+    height: 16px;
+  }
+
+  .profile_text {
+    text-decoration: none;
+    color: inherit;
+  }
+
+  &:hover {
+    color: $red-samf;
+  }
+}
+
+/* Mobile popup navigation */
+#mobile_popup_container {
+  position: fixed;
+  top: 60px;
+  left: 0;
+  width: 100vw;
+  height: calc(100vh - 60px);
+  min-height: 400px;
+  background: $navbar-bg-dark;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  z-index: 100;
+  @include for-desktop-up {
+    display: none;
+  }
+}
```

Hvorfor ikke bruke `$navbar-height-samf-three` her? Ser det er et gap på 4px fra 64px til 60px.
```suggestion
/* Mobile popup navigation */
#mobile_popup_container {
  position: fixed;
  top: $navbar-height-samf-three;
  left: 0;
  width: 100vw;
  height: calc(100vh - $navbar-height-samf-three);
  min-height: 400px;
  background: $navbar-bg-dark;
  color: white;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  z-index: 100;
  @include for-desktop-up {
    display: none;
  }
}
```

---

Fiksa.